docs/_articles/en/lwc/testing.md

```diff
@@ -7,22 +7,34 @@ Install [`sfdx-lwc-jest`](https://developer.salesforce.com/docs/component-librar
 
 ## Explore Your LWC Tests
 
-The LWC Tests sidebar provides several features. Here, you can see all your LWC tests at a glance. You can run one test method, the test methods in one file, or all your tests. You can view the results of your last test run. And you can jump from those results to the corresponding lines in your code. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an sfdx-project.json file in its root directory.)
```

Comment:
Copy edit: LWC tests provide a central location for LWC tests and allow you to run tests on the code you write. You can test methods in one file, view the results of the last tests run, and navigate to the corresponding lines of code directly from the tests run. To access the test sidebar, click the beaker icon in the View bar. If you don't see this icon, make sure the project you have open in VS Code contains an sfdx-project.json file in its root directory.